build.proj

```diff
@@ -36,23 +36,21 @@
           DependsOnTargets="$(TraversalBuildDependencies)" />
 
   <Target Name="BuildCustomTasks">
-      <MSBuild Projects="tools-local/Microsoft.DotNet.Build.Tasks.Local/Microsoft.DotNet.Build.Tasks.Local.builds" />
       <MSBuild Projects="tools-local/tasks/core-setup.tasks.builds" />
 
     <PropertyGroup>
       <DependencyModelTFM>netstandard1.3</DependencyModelTFM>
       <DependencyModelTFM Condition="'$(MSBuildRuntimeType)' != 'Core'">net451</DependencyModelTFM>
     </PropertyGroup>
     
-      <ItemGroup>
-        <CustomTaskDependencies Include="$(BuildToolsTaskDir)Newtonsoft.Json.dll" />
-        <CustomTaskDependencies Include="$(BuildToolsTaskDir)Microsoft.DotNet.PlatformAbstractions.dll" />
```

Comment:
`DependencyModel` depends on this assembly.  If we need to copy DependencyModel to `$(LocalBuildToolsTaskDir)`, I would assume this is still necessary.  Unless it is getting copied there by some other mechanism.